client/pages/signup.html

```diff
@@ -28,12 +28,41 @@ <h1> Sign Up </h1>
 		</form>
 	
 	</div>
+
+	<nav class="navbar navbar-default navbar-expand navbar-dark bg-dark">
+		<a href="#" class="navbar-brand">Torchbearer</a>
+		<button class="navbar-toggler" data-toggle="collapse" data-target="#navbarMenu">
+			<span class="navbar-toggler-icon"></span>
+	</button>
+	<div class="collapse navbar-collapse" id="navbarMenu">
+		<ul class="navbar-nav">
+			<li class="nav-item">
+				<a href="#" class="nav-link">Home</a>
+			</li>
+			<li class="nav-item">
+				<a href="#" class="nav-link">Board</a>
+			</li>
+			<li class="nav-item">
+				<a href="#" class="nav-link">Tasks</a>
+			</li>
+
+		</ul>
+
+	</div>
+
+
+</nav> 
 	
 	<!-- ---------------------------------------- Bootstrap ---------------------------------------- -->
```

Comment:
This section has two references to jquery and bootstrap. Not an error just redundant.